pkg/cmd/stack/translate.go

```diff
@@ -327,10 +327,10 @@ func getInitContainerCommandAndVolumeMounts(svc model.Service) ([]string, []apiv
 			if !addedDataVolume {
 				volumeMounts = append(volumeMounts, apiv1.VolumeMount{Name: volumeName, MountPath: "/data"})
 				if command == "" {
-					command = "chmod 777 /data/"
+					command = "chmod 777 /data/*"
```

Comment:
do we need to do `chmod +R` so it's applied to the subdirectories and files?

Reply:
No, we want to avoid that. If you modify the permissions of files created by the container, the container might fail. We do `*` because okteto creates first-level subfolders at this level